.github/workflows/buildcheck.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Build on PR
+
+on:
+  pull_request_target:
+    types: [opened, reopened, edited, labeled]
+
+jobs:
+  Manifest-Generator:    
```

Comment:
A more desciptive job name please - it is invoking a build.

No mixed case, all lower e.g. `job-name`